### PR TITLE
Tweak and run native code formatting script

### PR DESCRIPTION
**Pull request by nguerrera**
* Rename script to format-code.sh to match naming convention of other shell scripts in the repo.

* Make script work when executed from any working directory.

* Run script to mop up some minor formatting issues that crept in.

### CODE REVIEW

**Comment by nguerrera** [comment]
cc @sokket

File: src/Native/format-code.sh [hunk 1]
```diff
@@ -11,6 +11,8 @@ else
    exit 1
 fi
 
+cd $(dirname "$0")
```
Comment (stephentoub) [inline src/Native/format-code.sh]: Should we pushd/pop?

**Comment by stephentoub** [comment]
LGTM

File: src/Native/format-code.sh (hunk 1 above)
Comment (nguerrera) [inline src/Native/format-code.sh]: cd doesn't propagate back to parent process, so I don't think it matters.

**Pull request merged**